src/main/java/org/kuali/kfs/pdp/businessobject/PaymentGroup.java

```diff
@@ -1107,21 +1107,22 @@ public String getStreet() {
     }
     
     public void validateVendorIdAndCustomerInstitutionIdentifier() {
```

Comment:
I think the custom vendor-related methods in this file are being used by CU-specific pieces of code elsewhere. It might make more sense to customize those areas instead of this file, depending on how difficult it is to get those other pieces of code to work for non-vendor payment groups.